config/nrfconnect/chip-module/Kconfig.features

```diff
@@ -138,4 +138,18 @@ endif # SOC_SERIES_NRF53X
 
 endif # CHIP_DFU_OVER_BT_SMP
 
+if MPSL_FEM
+
+config MPSL_FEM_NRF21540_RUNTIME_PA_GAIN_CONTROL
+	bool
+	default y
+
+endif # MPSL_FEM
+
+config OPENTHREAD_DEFAULT_TX_POWER
+	int
+	default 20 if MPSL_FEM
+	default 3 if SOC_SERIES_NRF53X && !MPSL_FEM
```

Comment:
I think this now better fits `Kconfig.defaults` as you don't add any new feature sets, you just modify existing Kconfigs.